frontend-ssr/package-lock.json

```diff
@@ -0,0 +1,8758 @@
+{
```

Comment:
NextJS uses `yarn` so we should install dependencies this way. YARN is another package manager, similar to `npm`. Run `npm install -g yarn` to install it globally and then `yarn` inside the project to install de dependencies.

Also delete this file, instead we should have a `yarn-lock.json` or smth like that.

Reply:
The `package-lock.json` is still here. Did you forget to `push` the new changes?